frontend/src/components/NavDropdown.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
```

Comment:
Note: when Header component is broken down into smaller components, this file could be moved under a the new "header" directory will all header related components.